.circleci/config.yml

```diff
@@ -417,6 +417,14 @@ workflows:
             - test_211_jdk11_jvm
             - test_211_jdk8_js
             - test_211_jdk11_js
+            - test_212_jdk8_jvm
```

Comment:
- I would prefer `test_212` tests to be moved to the `require` section of `release_212` please

Reply:
here https://github.com/zio/zio/blob/master/.circleci/config.yml#L437

Reply:
here https://github.com/zio/zio/blob/master/.circleci/config.yml#L428

Reply:
Isn’t it better to stop releasing if any test fail? If we move it as you say, we might release 2.11 and not 2.12.

Reply:
Yeah not sure which strategy you prefer. I'll let you decide.

Reply:
It’s a mess when we release 2.11 and not 2.12 (happened a couple time in the past) so I’d really prefer checking all of them before releasing 2.11. 
@softinio can you confirm it’s ok with you?

Reply:
Even though it infringes my ocd 🤣I do agree with your point which I had not thought of. Thank you both. Approving.